package.json

```diff
@@ -31,16 +31,16 @@
     "gulp-svg-sprite": "^1.2.7",
     "gulp-util": "^3.0.6",
     "husky": "^0.10.1",
+    "jasmine-core": "^2.3.4",
     "json-loader": "^0.5.3",
     "karma": "^0.13.15",
     "karma-chrome-launcher": "^0.2.1",
     "karma-cli": "^0.1.1",
-    "karma-mocha": "^0.2.0",
-    "karma-sinon-chai": "^1.1.0",
+    "karma-jasmine": "^0.2.2",
```

Comment:
Basically, `jasmine === mocha + chai + sinon` with less verbose syntax and easier custom matchers